src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/JellyVerifier.java

```diff
@@ -0,0 +1,141 @@
+package io.jenkins.infra.repository_permissions_updater.hosting;
+
+import static java.util.regex.Pattern.CASE_INSENSITIVE;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.xml.XMLConstants;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import org.apache.commons.lang3.StringUtils;
+import org.kohsuke.github.GHContent;
+import org.kohsuke.github.GHContentSearchBuilder;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.PagedSearchIterable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Attr;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+public class JellyVerifier implements Verifier {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JellyVerifier.class);
+    private static final String INLINE_STYLE = "The jelly file %s contains an inline `<style>` tag";
+    private static final String INLINE_SCRIPT = "The jelly file %s contains a `<script>` tag with inline javascript.";
+    private static final String INLINE_SCRIPT_METHOD = "The jelly file %s potentially uses inline javascript attribute `%s`";
```

Comment:
Could these file paths be a link to GitHub?

Reply:
yes